PerfectDecimal/PerfectDecimal/PerfectDecimal.csproj

```diff
@@ -0,0 +1,9 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Should ImplicitUsings be enabled?

Reply:
I think we should leave implicit usings enabled. They can be disabled later depending on user demand.